src/mame/misc/xtom3d.cpp

```diff
@@ -409,20 +426,37 @@ ROM_START( xtom3d )
 	ROM_LOAD( "u20", 0x200000, 0x200000, CRC(452131d9) SHA1(f62a0f1a7da9025ac1f7d5de4df90166871ac1e5) )
 ROM_END
 
+
+ROM_START( pumpit1 )
+	ROM_REGION32_LE(0x20000, "pci:07.0", 0)
+//	ROM_LOAD( "bios.u22", 0x000000, 0x010000, CRC(574bb327) SHA1(c24484e9b304b9d570c5ead6be768f563d5c389f) )
+	ROM_LOAD( "bios.u22", 0x000000, 0x020000, CRC(f7c58044) SHA1(fd967d009e0d3c8ed9dd7be852946f2b9dee7671) )
+
+	ROM_REGION32_LE(0x1000000, "board1:game_rom", ROMREGION_ERASEFF )
+	ROM_LOAD( "piu10.u8",  0x000000, 0x200000, CRC(5911e31a) SHA1(295723b9b7da9e55b5dd5586b23b06355f4837ef) )
+
+	ROM_REGION(0x400000, "board3:sound_rom", ROMREGION_ERASEFF )
+	ROM_LOAD( "piu10.u9",  0x000000, 0x200000, CRC(9c436cfa) SHA1(480ea52e74721d1963ced41be5c482b7b913ccd2) )
+
+	DISK_REGION( PCI_IDE_ID":ide1:0:cdrom:image" )
```

Comment:
Two issues here about why I haven't directly committed this to main pc_i440bx:
1. don't have a clearance about whatsnew credits, will sort it hopefully soon (and have space for at least one commit here)
2. I can't map the cdrom no matter what, it's supposed to give me back a Voodoo error but instead it just abort retry fail (and it works if I map it directly thru `-cdrom1` ), is it not mapping because the DISK_REGION is too early for init?

Reply:
FWIW, Pump It Up Exceed 1 & 2 run on mid-2000s PC hardware (with OSs like Windows XP) but nice to see these oldest Pump It Up titles being emulated.

Reply:
They are not, there's a full world of things to do here ...

Reply:
My mistake then.